src/main/java/com/marksem/crm/entity/Ttransaction.java

```diff
@@ -0,0 +1,27 @@
+package com.marksem.crm.entity;
+
+import lombok.*;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Table(name = "ttransaction")
+@Data
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+public class Ttransaction extends BaseEntity implements Serializable {
```

Comment:
You can safely call this class `Transaction`, it's okay if it clashes with some default implementations